src/views/SystemIntake/Documents/DocumentsTable.tsx

```diff
@@ -0,0 +1,320 @@
+    prepareRow,
+    rows
+  } = useTable(
+    {
+      columns,
+      data: documents,
+      autoResetSortBy: false,
+      autoResetPage: false,
+      initialState: {
+        sortBy: useMemo(() => [{ id: 'uploadedAt', desc: true }], [])
+      }
+    },
+    useSortBy
+  );
+
+  const ConfirmDeleteModal = () => {
+    if (!fileToDelete) return null;
+    return (
+      <Modal isOpen={!!fileToDelete} closeModal={() => setFileToDelete(null)}>
+        <h3 className="margin-top-0 margin-bottom-0">
+          {t(
+            'technicalAssistance:documents.supportingDocuments.removeHeading',
+            {
+              documentName: fileToDelete.fileName
+            }
+          )}
+        </h3>
+
+        <p>
+          {t('technicalAssistance:documents.supportingDocuments.removeInfo')}
+        </p>
+
+        <Button
+          type="button"
+          onClick={() => {
+            deleteDocument({ variables: { id: fileToDelete.id } })
+              .then(() => {
+                showMessage(
+                  <Alert type="success">
+                    {t(
+                      'technicalAssistance:documents.supportingDocuments.removeSuccess',
+                      {
+                        documentName: fileToDelete.fileName
+                      }
+                    )}
+                  </Alert>
+                );
+              })
+              .catch(() => {
+                showMessage(
+                  <Alert type="error">
+                    {t(
+                      'technicalAssistance:documents.supportingDocuments.removeFail'
+                    )}
+                  </Alert>
+                );
+              });
+
+            setFileToDelete(null);
+          }}
+        >
+          {t(
+            'technicalAssistance:documents.supportingDocuments.removeDocument'
+          )}
+        </Button>
+
+        <Button
+          type="button"
+          className="margin-left-2"
+          unstyled
+          onClick={() => setFileToDelete(null)}
+        >
+          {t('technicalAssistance:documents.supportingDocuments.cancel')}
+        </Button>
+      </Modal>
+    );
+  };
+
+  return (
+    <div id="systemIntakeDocuments">
+      {/* {message}
+
+      <PageHeading className="margin-top-4 margin-bottom-1">
+        {t('intake:documents.title')}
+      </PageHeading>
+      <p className="margin-top-1 font-body-md line-height-body-5 tablet:grid-col-12 desktop:grid-col-8">
+        {t('intake:documents.tableDescription')}
+      </p>
+
+      <h4 className="margin-bottom-1 margin-top-5">
+        {t('intake:documents.tableTitle')}
+      </h4>
+
+      <Button
+        className="margin-bottom-1"
+        type="button"
+        onClick={() =>
+          history.push(`/system/${systemIntake.id}/documents/upload`)
+        }
+      >
+        {t(
+          `technicalAssistance:documents.${
+            documents.length ? 'addAnotherDocument' : 'addDocument'
+          }`
+        )}
+      </Button> */}
```

Comment:
This might be old/copied code that can be removed.  If it isn't, it might need a comment to explain it's future intended use